### PR TITLE
build(js): Bump to latest fork-ts-checker

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -334,8 +334,13 @@ let appConfig = {
     ...(SHOULD_FORK_TS
       ? [
           new ForkTsCheckerWebpackPlugin({
-            eslint: TS_FORK_WITH_ESLINT,
-            tsconfig: path.resolve(__dirname, './config/tsconfig.build.json'),
+            eslint: {
+              enabled: TS_FORK_WITH_ESLINT,
+              files: './sentry/**/*.{ts,tsx,js,jsx}',
```
Comment (billyvg) [inline webpack.config.js:339]: this doesn't include test, also shouldn't it be `src/sentry`?

File: yarn.lock [hunk 2]
```diff
@@ -15376,6 +15426,16 @@ universalify@^0.1.0:
   resolved "https://registry.yarnpkg.com/universalify/-/universalify-0.1.2.tgz#b646f69be3942dabcecc9d6639c80dc105efaa66"
   integrity sha512-rBJeI5CXAlmy1pV+617WB9J63U6XcazHHF2f2dbJix4XzpUF0RS3Zbj0FGIOCAva5P/d/GBOYaACQ1w+0azUkg==
 
+universalify@^1.0.0:
+  version "1.0.0"
+  resolved "https://registry.yarnpkg.com/universalify/-/universalify-1.0.0.tgz#b61a1da173e8435b2fe3c67d29b9adf8594bd16d"
+  integrity sha512-rb6X1W158d7pRQBg5gkR8uPaSfiids68LTJQYOtEUhoJUWBdaQHsuT/EUduxXYxcrt4r5PJ4fuHW1MHT6p0qug==
+
+universalify@^2.0.0:
```
Comment (billyvg) [inline yarn.lock:15434]: :thunking-face:

File: webpack.config.js (hunk 1 above)
Comment (EvanPurkhiser) [reply webpack.config.js:339]: 😬 yes it should.

Does anyone use the eslint checker with this?

Comment (billyvg) [reply webpack.config.js:339]: @priscilawebdev does

Comment (EvanPurkhiser) [reply webpack.config.js:339]: Looks like it won't actually run the older version of eslint we use on this newer version

**Review by billyvg — CHANGES_REQUESTED** [review]
via slack:

`Error: Cannot use current eslint version of 5.11.1, the minimum required version is 6.0.0`

Although we should probably upgrade this to match the version in getsentry in the meantime.

**Pull request closed without merging**